### PR TITLE
docs: update examples and fix whitespace

### PR DESCRIPTION
**Pull request by gunjjoshi**
## Description

> What is the purpose of this pull request?

This pull request:

-   addresses https://github.com/stdlib-js/stdlib/commit/6c8f019db8adb12466250957693edb9830496a7b#r146319283, https://github.com/stdlib-js/stdlib/commit/6c8f019db8adb12466250957693edb9830496a7b#r146319326 and https://github.com/stdlib-js/stdlib/commit/6c8f019db8adb12466250957693edb9830496a7b#r146319438, for [`math/base/special/logf`](https://github.com/stdlib-js/stdlib/tree/develop/lib/node_modules/%40stdlib/math/base/special/logf).
-   uses `discrete-uniform` instead of rounding numbers to generate integers.

## Related Issues

> Does this pull request have any related issues?

None.

## Questions

> Any questions for reviewers of this pull request?

No.

## Other

> Any other information relevant to this pull request? This may include screenshots, references, and/or implementation notes.

No.

## Checklist

> Please ensure the following tasks are completed before submitting this pull request.

-   [x] Read, understood, and followed the [contributing guidelines][contributing].

* * *

@stdlib-js/reviewers

[contributing]: https://github.com/stdlib-js/stdlib/blob/develop/CONTRIBUTING.md

### CODE REVIEW

File: lib/node_modules/@stdlib/math/base/special/logf/README.md [hunk 1]
```diff
@@ -66,17 +66,16 @@ v = logf( 2.0, -4.0 );
 <!-- eslint no-undef: "error" -->
 
 ```javascript
-var randu = require( '@stdlib/random/base/randu' );
-var roundf = require( '@stdlib/math/base/special/roundf' );
+var discreteUniform = require( '@stdlib/random/base/discrete-uniform' );
```
Comment (kgryte) [inline lib/node_modules/@stdlib/math/base/special/logf/README.md]: @gunjjoshi Let's use `random/array/discrete-uniform` here and for `examples/index.js`.

Comment (gunjjoshi) [reply lib/node_modules/@stdlib/math/base/special/logf/README.md]: Done!

File: lib/node_modules/@stdlib/math/base/special/logf/README.md [hunk 2]
```diff
@@ -66,18 +66,20 @@ v = logf( 2.0, -4.0 );
...
-var b;
-var x;
+var opts = {
+    'dtype': 'float32'
+};
+
+var x = randu( 100, 0, 100, opts );
+var b = randu( 100, 0, 5, opts );
+
 var i;
```
Comment (kgryte) [inline lib/node_modules/@stdlib/math/base/special/logf/README.md]:
```suggestion
```

File: lib/node_modules/@stdlib/math/base/special/logf/examples/index.js [hunk 3]
```diff
@@ -18,16 +18,18 @@
...
-var b;
-var x;
+var opts = {
+	'dtype': 'float32'
+};
+
+var x = randu( 100, 0, 100, opts );
+var b = randu( 100, 0, 5, opts );
+
 var i;
```
Comment (kgryte) [inline lib/node_modules/@stdlib/math/base/special/logf/examples/index.js]:
```suggestion
```

The reason for this nit is that we're trying to get our examples to a state of uniformity such that we can use code translation to convert to and from ES2015+ in web docs.

**Review by kgryte — APPROVED** [review]
LGTM. Thanks, @gunjjoshi!

**Pull request merged**